README.md

```diff
@@ -21,13 +25,22 @@ To set up Selfauth, create a folder on your webserver and add the files in this
     ... where `https://example.com/auth/` is the URL you installed Selfauth to.
     (The exact location of your HTML `<head>` could be hidden in your CMS. Look for help in their documentation. Setting a HTTP Link header like `Link: <https://example.com/auth/>; rel="authorization_endpoint"` should work too.)
 
-4. Go to a place to login with IndieAuth and enter your URL. (IndieAuth.com has a test-form on the frontpage. Enter your URL and it will discover your new endpoint. Click on that button. Other IndieAuth clients will redirect you to your Selfauth immediately.)
+You can delete the file `setup.php` if you want, but this is optional. It will not be able to save a new password for you once the setup is completed.
+
+
+## Changing your password
+
+To change your password, make sure the `setup.php` file is in place again and delete `config.php`. Then follow the steps under [Setup](#setup) again.
+
+
+## How it works
+
+On a (Web)App which supports [IndieAuth](https://indieweb.org/IndieAuth), you can enter your personal URL. The App will detect Selfauth as Authorization Endpoint and redirect you to it. After you enter your password in Selfauth, you are redirected back to the App with a code. The App will verify the code with Selfauth and logs you in as your personal URL.
 
-5. Fill in your password and click login!
+To test it, you can go to an App that supports IndieAuth and enter your personal URL. [IndieAuth.com](https://indieauth.com/) has a test-form on the frontpage. If you also link to your social media accounts using `rel="me"`, IndieAuth.com might show you a list of buttons, on which you can click the one that has your Selfauth URL on it.
```

Comment:
Last sentence here is really confusing.  Suggest splitting it to two sentences. 

IndieAuth.com might show you a list of buttons.  To use selfauth, click on the one that ...